openedx/core/djangoapps/content_libraries/tests/test_views_lti.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for LTI views.
+"""
+
+from django.test import TestCase
+from django.urls import reverse
+
+from openedx.core.djangoapps.content_libraries.constants import PROBLEM
+from openedx.core.djangoapps.content_libraries.views import LibraryBlockLtiUrlView
+
+from .base import URL_LIB_LTI_JWKS
+from .base import skip_unless_cms, ContentLibrariesRestApiTest
+
+
+
+@skip_unless_cms
+class LtiToolJwksViewTest(TestCase):
+    """
+    Test JWKS view.
+    """
+
+    def test_when_no_keys_then_return_empty(self):
+        """
+        Given no LTI tool in the database.
+        When JWKS requested.
+        Then return empty
+        """
+        response = self.client.get(URL_LIB_LTI_JWKS)
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(response.content, '{"keys": []}')
+
+
+class LibraryBlockLtiUrlViewTest(ContentLibrariesRestApiTest):
```

Comment:
Do you think it worth covering the "not found" scenario?

Reply:
I think that could be a good scenario as well.